src/controllers/controller.polarArea.js

```diff
@@ -123,7 +123,12 @@ module.exports = function(Chart) {
 			var meta = me.getMeta();
 			var opts = chart.options;
 			var arcOpts = opts.elements.arc;
+			var dataset = me.getDataset();
 			var minSize = Math.min(chartArea.right - chartArea.left, chartArea.bottom - chartArea.top);
+			var start = opts.startAngle || 0;
+			var starts = [];
+			var angles = [];
```

Comment:
I changed my previous comment a bit too late: 

```
var starts = me._starts = [];
var angles = me._angles = [];
```

And lines 148-149 can be removed :)